### PR TITLE
Reindent some files.

### PR DESCRIPTION
**Pull request by scrosby**
## Changes proposed in this PR

- Reindent some files.
- 
- 

## Why are we making these changes?
- Makes the code look nicer.

### CODE REVIEW

File: scheduler/src/cook/mesos/api.clj [hunk 1]
```diff
@@ -2205,7 +2205,7 @@
     (if (nil? pool)
       (let [pool-limits (get-pool-limits get-limit-fn db user)]
         (cond-> response
-          (not (empty? pool-limits)) (assoc :pools pool-limits)))
+                (not (empty? pool-limits)) (assoc :pools pool-limits)))
```
Comment (shamsimam) [inline scheduler/src/cook/mesos/api.clj]: `cond->` conditions and clauses should be indented one level. You can change Cursive settings to fix this.

File: scheduler/src/cook/mesos/api.clj [hunk 2]
```diff
@@ -2705,8 +2705,8 @@
                                  (not (string? body)))
                             (coll? body))]
       (cond-> resp
-        json-response (res/content-type "application/json")
-        (and json-response body) (assoc :body (streaming-json-encoder body))))))
+              json-response (res/content-type "application/json")
```
Comment (shamsimam) [inline scheduler/src/cook/mesos/api.clj]: `cond->` conditions and clauses should be indented one level. You can change Cursive settings to fix this.

File: scheduler/src/cook/mesos/api.clj [hunk 3]
```diff
@@ -815,8 +815,8 @@
   (->> datasets
        (map (fn [{:keys [dataset partitions]}]
               (cond-> {:dataset (walk/stringify-keys dataset)}
-                partitions
-                (assoc :partitions (into #{} (walk/stringify-keys partitions))))))
+                      partitions
```
Comment (shamsimam) [inline scheduler/src/cook/mesos/api.clj]: `cond->` conditions and clauses should be indented one level. You can change Cursive settings to fix this.

**Comment by shamsimam** [comment]
Here is a clojure style guide which mentioned how to handle `cond->`: https://github.com/bbatsov/clojure-style-guide#body-indentation

**Comment by pschorf** [comment]
Do we want to merge or close this?

**Comment by shamsimam** [comment]
I would like to merge this after the `cond->` changes.

File: scheduler/src/cook/mesos/api.clj (hunk 1 above)
Comment (scrosby) [reply scheduler/src/cook/mesos/api.clj]: Change reverted.

File: scheduler/src/cook/mesos/api.clj (hunk 2 above)
Comment (scrosby) [reply scheduler/src/cook/mesos/api.clj]: Reverted

File: scheduler/src/cook/mesos/api.clj (hunk 3 above)
Comment (scrosby) [reply scheduler/src/cook/mesos/api.clj]: Reverted.

**Comment by scrosby** [comment]
Reverted the problematic changes and fixed another. Should be good to merge.

**Pull request merged**